gpjax/gps.py

```diff
@@ -203,15 +203,24 @@ def predict(
             should be evaluated at. The mean function's value at these points is
             then returned.
         """
-        gram = self.kernel.gram
         jitter = get_defaults()["jitter"]
 
+        # Unpack mean function and kernel
+        mean_function = self.mean_function
+        kernel = self.kernel
```

Comment:
I don't see the advantage of this? It's only going to make stepping back through the code for new users harder.